trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_wing_induced_velocity.py

```diff
@@ -14,82 +13,79 @@
-    inv_root_beta[mach>1] = 1/np.sqrt(mach[mach>1]**2-1) 
-    inv_root_beta = np.atleast_3d(inv_root_beta)
-     
-    XAH   = np.atleast_3d(VD.XAH*inv_root_beta) 
-    YAH   = np.atleast_3d(VD.YAH*ones) 
-    ZAH   = np.atleast_3d(VD.ZAH*ones) 
-    XBH   = np.atleast_3d(VD.XBH*inv_root_beta) 
-    YBH   = np.atleast_3d(VD.YBH*ones) 
-    ZBH   = np.atleast_3d(VD.ZBH*ones) 
-
-    XA1   = np.atleast_3d(VD.XA1*inv_root_beta)
-    YA1   = np.atleast_3d(VD.YA1*ones)
-    ZA1   = np.atleast_3d(VD.ZA1*ones)
-    XA2   = np.atleast_3d(VD.XA2*inv_root_beta)
-    YA2   = np.atleast_3d(VD.YA2*ones)
-    ZA2   = np.atleast_3d(VD.ZA2*ones)
-
-    XB1   = np.atleast_3d(VD.XB1*inv_root_beta)
-    YB1   = np.atleast_3d(VD.YB1*ones)
-    ZB1   = np.atleast_3d(VD.ZB1*ones)
-    XB2   = np.atleast_3d(VD.XB2*inv_root_beta)
-    YB2   = np.atleast_3d(VD.YB2*ones)
-    ZB2   = np.atleast_3d(VD.ZB2*ones) 
-    
-    XC_TE   = np.atleast_3d(VD.XC_TE*inv_root_beta)
-    YC_TE   = np.atleast_3d(VD.YC_TE*ones)
-    ZC_TE   = np.atleast_3d(VD.ZC_TE*ones)    
-    XA_TE   = np.atleast_3d(VD.XA_TE*inv_root_beta)
-    YA_TE   = np.atleast_3d(VD.YA_TE*ones)
-    ZA_TE   = np.atleast_3d(VD.ZA_TE*ones)
-    XB_TE   = np.atleast_3d(VD.XB_TE*inv_root_beta)
-    YB_TE   = np.atleast_3d(VD.YB_TE*ones)
-    ZB_TE   = np.atleast_3d(VD.ZB_TE*ones) 
-    
-    XC    = np.atleast_3d(VD.XC*inv_root_beta)
-    YC    = np.atleast_3d(VD.YC*ones) 
-    ZC    = np.atleast_3d(VD.ZC*ones)  
-    n_w   = VD.n_w
-
-    theta_w = np.atleast_3d(theta_w)   # wake model, use theta_w if setting to freestream, use 0 if setting to airfoil chord like
+    n_cp     = n_sw*n_cw
+    n_w      = VD.n_w
+
+    # Control points from the VLM 
+    XAH   = np.atleast_2d(VD.XAH*1.) 
+    YAH   = np.atleast_2d(VD.YAH*1.) 
+    ZAH   = np.atleast_2d(VD.ZAH*1.) 
+    XBH   = np.atleast_2d(VD.XBH*1.) 
+    YBH   = np.atleast_2d(VD.YBH*1.) 
+    ZBH   = np.atleast_2d(VD.ZBH*1.) 
+    XA1   = np.atleast_2d(VD.XA1*1.) 
+    YA1   = np.atleast_2d(VD.YA1*1.) 
+    ZA1   = np.atleast_2d(VD.ZA1*1.) 
+    XB1   = np.atleast_2d(VD.XB1*1.) 
+    YB1   = np.atleast_2d(VD.YB1*1.) 
+    ZB1   = np.atleast_2d(VD.ZB1*1.)    
+    XA2   = np.atleast_2d(VD.XA2*1.) 
+    YA2   = np.atleast_2d(VD.YA2*1.) 
+    ZA2   = np.atleast_2d(VD.ZA2*1.) 
+    XB2   = np.atleast_2d(VD.XB2*1.) 
+    YB2   = np.atleast_2d(VD.YB2*1.) 
+    ZB2   = np.atleast_2d(VD.ZB2*1.)       
+    XC    = np.atleast_2d(VD.XC*1.)
+    YC    = np.atleast_2d(VD.YC*1.) 
+    ZC    = np.atleast_2d(VD.ZC*1.)  
+    XA_TE = np.atleast_2d(VD.XA_TE*1.)  
+    XB_TE = np.atleast_2d(VD.XB_TE*1.)  
+    ZA_TE = np.atleast_2d(VD.ZA_TE*1.)  
+    ZB_TE = np.atleast_2d(VD.ZB_TE*1.)      
+    
+    # supersonic corrections
+    beta_2     = 1-mach**2
+    beta_2_1d  = beta_2*1.
```

Comment:
doesn't look like beta_2_1d is used anywhere

Reply:
Good catch. lol I deleted that right before you commented.